.github/workflows/localPa11y.yml

```diff
@@ -0,0 +1,43 @@
+name: localPa11yci
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+ 
+jobs:
+  build:
+ 
+    runs-on: ubuntu-latest
+ 
+    strategy:
+      matrix:
+        node-version: [v20.16.0]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+ 
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm install -g pa11y-ci
+    - run: npm install wait-on
+    - run: npm start & npx wait-on http://localhost:3000
+    - run: pa11y-ci --config ./localPa11y.json 2>&1 | tee pa11y_output.txt
+        
+    - name: Read pa11y_output file.
+      id: pa11y_output
+      uses: juliangruber/read-file-action@v1
+      with:
+        path: ./pa11y_output.txt
+
+    - name: Check for pa11y failures.
+      if: contains(steps.pa11y_output.outputs.content, 'Errors in')
+      run: |
+        echo "::error::The site is failing accessibility tests. Please review the comment in the pull request or the pa11y-ci step in the workflow for details."
```

Comment:
Please change the error message as there is no comment left in the PR if the test fails